ray-operator/config/samples/ray-cluster.heterogeneous.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     # An unique identifier for the head node and workers of this cluster.
   name: raycluster-heterogeneous
 spec:
-  rayVersion: '1.0.1'
+  rayVersion: '1.6.0'
```

Comment:
I'm not sure, but the comment and image in ray-cluster.mini.yaml was rayVersion 1.6.0

Reply:
This is fine. Currently, we reserve this field and don't consider the compatibility across version.

Reply:
I got it. Thanks :)